src/renderer/features/playlists/components/playlist-detail-content.tsx

```diff
@@ -221,14 +234,12 @@ export const PlaylistDetailContent = ({ tableRef }: PlaylistDetailContentProps)
                     suppressLoadingOverlay
                     suppressRowDrag
                     columnDefs={columnDefs}
-                    getRowId={(data) => {
-                        // It's possible that there are duplicate song ids in a playlist
-                        return `${data.data.id}-${data.data.pageIndex}`;
-                    }}
+                    getRowId={getId}
                     rowClassRules={rowClassRules}
                     rowData={playlistSongData}
                     rowHeight={60}
                     rowSelection="multiple"
+                    shouldUpdateSong={server?.type === ServerType.JELLYFIN}
```

Comment:
Does this not work for Navidrome?

Reply:
Outdated, but this was because the Jellyfin implementation can rely on the fact there were no duplicates, whereas Navidrome cannot

Reply:
Oh my bad, I was still reviewing without refreshing your changes :)